hc/front/views.py

```diff
@@ -393,6 +401,31 @@ def add_email(request):
     return render(request, "integrations/add_email.html", ctx)
 
 
+@login_required
+def add_telegram(request):
+    ctx = {"page": "channels"}
+    if "TOKEN" in os.environ:
+        token = os.environ.get("TOKEN")
+        updater = Updater(token=token)
+
+        def start(bot, update):
+            bot.send_message(chat_id=update.message.chat_id,
+                             text=str("Hello, Welcome to healthchecksbot. " + " Your chat ID is " +
+                                      str(update.message.chat_id)))
+        start_handler = CommandHandler('start', start)
+        # add handlers
+        updater.dispatcher.add_handler(start_handler)
+        updater.start_polling()
+        return render(request, 'integrations/add_telegram.html', ctx)
```

Comment:
This return statement however, may not be necessary.

Reply:
Okay let me get rid of it

Reply:
Thank you for that feedback. I have implemented the changes